README.rst

```diff
@@ -2,20 +2,20 @@
 INSTALL
 #######
 
-To install the project from the development::
+To run the demo project for testing::
 
     $ git clone git://github.com/jojax/django-js-error-hook.git
     $ cd django-js-error-hook
-    $ virtualenv env
-    $ env/bin/python setup.py develop
+    $ python3 -m venv env
+    $ source env/bin/activate
+    $ python setup.py develop
     $ cd demo
-    $ ../env/bin/python demo/setup.py develop
```

Comment:
I'd lilke to keep the demo installable in the venv.

Reply:
It is installable in the venv.

Reply:
The line ```source env/bin/activate``` means that all the subsequent python command calls are using the python from the virtual env.

Reply:
My point is that a lot of changes that you've made here are not about porting to Django 2.0 but are removing features that were implemented in order for people to install the demo in a venv and run it without having to move in the demo folder.

Reply:
What features?

Reply:
It is all still running in the venv. The venv is being activated in line 10. The line "cd" is still moving the user into the demo folder, like before.

Reply:
Ok. yeah, I may have misunderstood what the purpose with some of this was. I received a lot of error messages and the instructions gave me instructions that didn't quite work, so I tried to update as good as I could. If there is anything missing now that used to be there, the easiest is probably if you change it to work again.